packages/webcert/src/page/CertificatePage.tsx

```diff
@@ -72,12 +72,12 @@ const CertificatePage: React.FC = () => {
     <Root>
       <CertificateDeletedModal routedFromDeletedCertificate={routedFromDeletedCertificate} />
       <NoFlexGrow>
-        <SystemBanners />
         <AppHeader
           logo={logo}
           alt={'Logo Webcert'}
           primaryItems={[<WebcertHeaderUser />, <WebcertHeaderUnit />]}
           secondaryItems={secondaryItems}
+          banners={[<SystemBanners />]}
```

Comment:
Why pass component as prop instead of including it in `AppHeader`? I think that would make more sense since all pages will have header and system banners components.

Reply:
The AppHeader is built to be used by different applications, and is therefore placed in common. The component SystemBanners is unique for the application which is why it's placed in webcert. No components from webcert should be used in common.